### ISSUE TITLE
background-image gradients are lost if background-image has a url defined as the "background"

### ISSUE THREAD

**Issue by AlexKey** [jakubpawlowicz/clean-css#466]
Take the following CSS:

```CSS
.test {
    background: url("http://placehold.it/100x100");
    background-repeat: repeat;
    background-image: linear-gradient(rgba(44, 44, 44, 0.1), rgba(44, 44, 44, 0.5)), url("http://placehold.it/100x100");
    width: 100px;
    height: 100px;
}
```

This will result in:

```CSS
.test{background:url(http://placehold.it/100x100);background:url(http://placehold.it/100x100);width:100px;height:100px}
```

Note, the linear-gradient is lost.

If you remove the url on the background e.g. the following

```CSS
background-image: linear-gradient(rgba(44, 44, 44, 0.1), rgba(44, 44, 44, 0.5)), url("http://placehold.it/100x100");
```

becomes:

```CSS
background-image: linear-gradient(rgba(44, 44, 44, 0.1), rgba(44, 44, 44, 0.5))
```

It does not occur.

It is probably not the most sensible CSS, however it is interpreted in the [browser as expected](http://jsfiddle.net/b7pLgtw4/), whereas clean-css gives you a surprise :-)

**Comment by jakubpawlowicz** [comment]
That's surprising indeed!

**Comment by jakubpawlowicz** [comment]
Fixed in 3.1.2!

**Comment by AlexKey** [comment]
Thanks @jakubpawlowicz ! :-)
